src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/config.jelly

```diff
@@ -2,30 +2,43 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
     
 <f:section title="Generate Test Artifacts">
-  	 <f:entry field="pdfReportChkBx">	
-    	<f:checkbox title="PDF test report " name="pdfReportChkBx" checked="${instance.pdfReportChkBx}"/>	
+    <f:optionalBlock name="pdfReportChkBx" field="pdfReportChkBx" title="PDF test report" checked="${instance.isPdfChecked}">
+  	 <f:entry field="pdfReportFilePath" title="File location ">	
```

Comment:
"File location" is fine. "File path" is another possibility.

Reply:
Changed to **File path :**